src/handlers/comment/handlers/assign.ts

```diff
@@ -13,17 +13,23 @@ export const assign = async (body: string) => {
   const config = getBotConfig();
 
   const payload = _payload as Payload;
+  const organization = payload.organization;
+
   logger.info(`Received '/assign' command from user: ${payload.sender.login}, body: ${body}`);
   const issue = (_payload as Payload).issue;
   if (!issue) {
     logger.info(`Skipping '/assign' because of no issue instance`);
     return "Skipping '/assign' because of no issue instance";
   }
 
+  if (!organization?.id) {
+    logger.info(`Skipping '/assign' because there's no organization details`);
+    return "Skipping '/assign' because of no issue instance";
```

Comment:
Pls make the error message more meaningful , like in the log message above: `Skipping '/assign' because there's no organization details `